### ISSUE TITLE
Doesn't work with "eslint": "^8.57.1"

### ISSUE THREAD

**Issue by matiasgarcia** [mizdra/eslint-interactive#389]
Hi!

I am doing  `eslint-interactive --config .eslintrc.js`, but it seems to be ignoring the errors:

```
🕛  Linting...
✔ Linting done.

💚 No error found. 
```

Example of `eslint`

```
/Users/matias/projects/services-nuvem-pago-argentina/src/application-gateway/presentation/tcp/gateway.controller.spec.ts
  173:31  error  Insert `⏎·······`  prettier/prettier

/Users/matias/projects/services-nuvem-pago-argentina/src/dlocal-webhooks/application/dtos/notifications/dlocal/installments-responsible-update-notifications/installments-responsible-update-payload.dto.ts
  20:2  error  Insert `⏎`  prettier/prettier

/Users/matias/projects/services-nuvem-pago-argentina/src/dlocal-webhooks/application/dtos/notifications/dlocal/refund-chargeback.dto.ts
  2:10  error  'Transform' is defined but never used  @typescript-eslint/no-unused-vars

/Users/matias/projects/services-nuvem-pago-argentina/src/dlocal-webhooks/application/services/app-webhook.service.ts
  5:3  error  'HttpException' is defined but never used                 @typescript-eslint/no-unused-vars
  6:3  error  'InternalServerErrorException' is defined but never used  @typescript-eslint/no-unused-vars

/Users/matias/projects/services-nuvem-pago-argentina/src/recipient/domain/recipient/repository.interface.ts
  1:10  error  'IdentityValidationStatus' is defined but never used  @typescript-eslint/no-unused-vars

/Users/matias/projects/services-nuvem-pago-argentina/src/recipient/infra/data/recipient/database-recipient.repository.spec.ts
  2:10  error  'IdentityValidationStatus' is defined but never used  @typescript-eslint/no-unused-vars

/Users/matias/projects/services-nuvem-pago-argentina/src/recipient/infra/data/recipient/database-recipient.repository.ts
  3:3  error  'IdentityValidationStatus' is defined but never used  @typescript-eslint/no-unused-vars

/Users/matias/projects/services-nuvem-pago-argentina/src/recipient/infra/services/payment-provider/d-local/d-local-request.builder.ts
  9:10  error  'ErrorLog' is defined but never used  @typescript-eslint/no-unused-vars

/Users/matias/projects/services-nuvem-pago-argentina/src/recipient/infra/services/payment-provider/d-local/marketplace/marketplace.factory.spec.ts
  28:43  error  Replace `·providerApiVersion:·ProviderApiVersion.v1` with `⏎······providerApiVersion:·ProviderApiVersion.v1,⏎···`  prettier/prettier
```

**Comment by mizdra** [comment]
Thank you for sending the issue!

This issue lacks reproduction steps. It is difficult to determine whether this is a bug.

Could you please provide minimal reproduction and reproduction steps? Open the following Playground and click the fork button to easily create minimal reproduction.

- https://github.com/mizdra/eslint-interactive?tab=readme-ov-file#playground

**Comment by mizdra** [comment]
I am closing this issue because I cannot reproduce the problem.
